<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DND_GM_Helper</class>
 <widget class="QWidget" name="DND_GM_Helper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="Racetxtin">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Race">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Race :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Rand Race</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="outNPCinfo">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>371</width>
     <height>561</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="butPrintNPCinfo">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>5</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Print NPC info</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Class :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Classtxtin">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Rand Class</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Name :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Nametxtin">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Rand Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Last Name : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LastNametxtin">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Rand Last Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>66</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Sexe :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butMale">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>51</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butnonbinary">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Non Binary</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Sexetxtin">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>66</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Size :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Sizetxtin">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="butRandsize">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Rand Size (with race)</string>
   </property>
  </widget>
  <widget class="QLabel" name="Age">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Age :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="agetxtin">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Rand Age (with race)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butCOmandlinemode">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>572</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Command line Mod</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Level : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leveltxtin">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>220</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="butrandlevel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Rand Level</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>221</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;| XP : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="xptxtin">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="butRandXP">
   <property name="geometry">
    <rect>
     <x>346</x>
     <y>220</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Rand XP</string>
   </property>
  </widget>
  <widget class="QLabel" name="HP">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;HP : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="HPtxtin">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="butCalculatehp">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate HP (with class)</string>
   </property>
  </widget>
  <widget class="QLabel" name="Aliment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Aliment : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ddAlimetin">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>280</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Lawful Good</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Neutral Good</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chaotic Good</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lawful Neutral</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Neutral Neutral</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chaotic Neutral</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lawful Evil</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Neutral Evil</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chaotic Evil</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>280</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Rand aliment</string>
   </property>
  </widget>
  <widget class="QLabel" name="Divinity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Divinity : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ddDivinity">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>251</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>No divinity</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="butRandDivinity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Rand Divinity</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butDivinitybyaliment">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Divinity by aliment</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
